pkgs/applications/networking/feedreaders/feeds/default.nix

```
@@ -0,0 +1,108 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+
+, appstream
+, gobject-introspection
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook
+
+, glib
+, gtk3
+, libhandy
+, webkitgtk
+, python3
+}:
+let
+  listparser = python3.pkgs.buildPythonPackage rec {
+    pname = "listparser";
+    version = "0.18";
+
+    src = python3.pkgs.fetchPypi {
+      inherit pname version;
+      sha256 = "0hdqs1mmayw1r8yla43hgb4d9y3zqs5483vgf8j9ygczkd2wrq2b";
+    };
+
+    propagatedBuildInputs = with python3.pkgs; [
+      requests
+      six
+    ];
+
+    doCheck = false;
+
+    meta = with lib; {
+      description = "A parser for subscription lists";
+      homepage = "https://github.com/kurtmckee/listparser";
+      license = licenses.lgpl3;
+      maintainers = [
+        maintainers.pbogdan
+      ];
+      platforms = platforms.linux;
+    };
+  };
+in
+python3.pkgs.buildPythonApplication rec {
+  pname = "feeds-unstable";
+  version = "22-06-2020";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "World";
+    repo = "gfeeds";
+    rev = "21c3dc7eb4f78493f13f62455db59184fc8fa33a";
+    sha256 = "0zq45arw0jnx4zq11bg5q8j7cmwh6bqv15xhpdid2fnwrs4x2m8r";
+  };
+
+  format = "other";
+
+  nativeBuildInputs = [
+    appstream
+    gobject-introspection
```

```suggestion

    glib # for glib-compile-schemas
    gobject-introspection
```

glib is needed at build time as well: https://gitlab.gnome.org/World/gfeeds/-/blob/d8bd1edb496cb9c397094278527a58bd759460ac/meson_post_install.py#L14